prerequisites.rst

```diff
@@ -1,35 +1,280 @@
+  Hard Disk Drives (HDD), Disk-on-Module [3]_, CompactFlash [4]_ or thumb drives (USB),
+  the Solid State Disks (SDD [2]_) must be managed.
+
+Data drive storage (DDS)
+  The storage disk drive(s) where the user data will be stored. **Cannot be 
+  the same of the system drive(s) and sizes will depend of the usage of the stored user data**.
+  Shared resources are managed over partitions, which is discussed in the software
+  requirements topic below. The recommendation for the Data drives must be
+  Solid State Disks (SSD [2]_ disks) for best performance or spinning 
+  Hard Disk Drives (HDD) for better device durability over time.
+
+The memory (RAM)
+^^^^^^^^^^^^^^^^
+
+Enough RAM is vital to maintaining peak performance. There are several combinations
+of installation sizes but the **recommendation is at least 4GiB for novice administrators**.
+
+For best practice you should have 8GiB of RAM for basic operations in default installation,
+and no matter the size, RAM must be configured in Dual Channel mode [8]_ 
+inclusivelly if there are little amount of, it will improve performance.
+
+Unless other NAS systems the |omv| system can run in at least 1GiB of RAM, but
+of course will need expertise.
+
+The communication card (NIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A NAS system as means is a network oriented system, of course |omv| is the
+most flexible NAS system in the world and allows multiple ways of communication, 
+this is becouse |omv| offers several modes of shared resources like RSYNC that 
+do not depends on networking. But there are few key points:
+
+- The NAS write speed to storage is limited to the speed of your NAS computer NIC card.
+- The NAS write speed to storage is limited to the speed of your network.
+- Speed of writing to the storage depends on ratio of the number of users accessing it.
+
+Full-duplex NICs are recommended, half-duplex NICs typically offer 10/100 Mbit/s
+and storage drives far exceed those speeds but are of course supported by |omv|.
+
+Any supported NIC vendor is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
+The architecture (CPU)
+^^^^^^^^^^^^^^^^^^^^^^
+
+We have already denoted in the requirements table the supported architectures or
+processors (which reduces and determines which computers can be used) but this is
+really a reference since it is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
+Any computer system supported by the common requirements of Debian operating
+system could be an |omv| system installation target. **Currently supported are
+AMD64 (x86/x86-64 64bits), i386 (x86 32bits), ARMv7+ARMv8 (arm64 64bits armel/armhf 32bits)**.
+The i386 version stopped its official support in version 6 although it is still 
+being released by the project due the nature of the |omv| technology.
```

Comment:
```suggestion
The i386 version is no longer officially supported since version 6, although it is still published by the project due to the nature of |omv| technology.
```

Reply:
commit with 70f443af661407cfca3c794da2723e26d987bf2e